web/server/vue-cli/package.json

```diff
@@ -27,21 +27,21 @@
   },
   "dependencies": {
     "@mdi/font": "^5.9.55",
-    "codechecker-api": "file:../../api/js/codechecker-api-node/dist/codechecker-api-6.39.0.tgz",
     "chart.js": "^2.9.4",
     "chartjs-plugin-datalabels": "^0.7.0",
+    "codechecker-api": "file:../../api/js/codechecker-api-node/dist/codechecker-api-6.39.0.tgz",
     "codemirror": "^5.60.0",
     "date-fns": "^2.20.0",
     "highlight.js": "^10.7.2",
-    "jsplumb": "^2.15.5",
     "js-cookie": "^2.2.1",
+    "jsplumb": "^2.15.5",
     "lodash": "^4.17.21",
     "marked": "^2.0.1",
     "splitpanes": "^2.3.6",
     "vue": "^2.6.11",
-    "vuetify": "^2.3.3",
     "vue-chartjs": "^3.5.1",
     "vue-router": "^3.5.1",
+    "vuetify": "^2.3.3",
     "vuex": "^3.6.2"
   },
```

Comment:
These changes are due to the `make package` command I guess. Should I delete this and `package-lock.json` file and again push it to the master?

Reply:
I don't think it was the `make package` target which did these changes. Maybe a wrong rebase or you branched from an earlier version of your master branch. Yes, please remove these changes because these are unnecessary changes. Also remove the changes in the package-lock.json file.